modules/netstat/win_test.go

```diff
@@ -0,0 +1,194 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// Contributor: Rob Murtha robmurtha@gmail.com [:robmurtha]
+package netstat
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
```

Comment:
Is it possible for us just to use standard functions here or is it a pain vs. importing this dependency?

I think it's OK either way but it's probably preferred we don't introduce new dependencies unless we have to.